js/examples/next-openai-telemetry-app/instrumentation.ts

```diff
@@ -1,10 +1,19 @@
 import { registerOTel } from "@vercel/otel";
 import { diag, DiagConsoleLogger, DiagLogLevel } from "@opentelemetry/api";
+import { SimpleSpanProcessor } from "@opentelemetry/sdk-trace-base";
+import { OpenInferenceProtoTraceExporter } from "../../packages/openinference-trace-exporter/src/OpenInferenceProtoTraceExporter";
```

Comment:
might need to think about package structure here, when i just try to import from /src next gets mad about having the grpc exporter in there... not sure what would happen when fully built - could subpackage or just punt on grpc for now

Reply:
that will be a blocker for arize, maybe we just need to do some use server stuff for next, will look into it

Reply:
there's a guard against this I believe in next itself

Reply:
switched to span processor so no longer relavant